deploy/backplane/lpsre/hypershift/management-cluster/10-lpsre.SubjectPermission.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-lpsre-management-cluster
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - allowFirst: true
+    clusterRoleName: admin
+    namespacesAllowedRegex: '(^openshift-package-operator$)'
+  - allowFirst: true
+    clusterRoleName: backplane-lpsre-management-cluster-project
+    namespacesAllowedRegex: '(^ocm-staging-.*|^ocm-production-.*|^hc-mc-.*|^openshift-package-operator$)'
```

Comment:
What are the `^hc-ms-.*` namespaces?

Why include environment in the ocm regex and not leave it open?  This misses integration.

Reply:
I wanted to grant access to HCP namespaces, but I got the wrong prefixes.
Changed to `cluster-.*` which should make more sense.

Reply:
HCP NS are ocm-.*

Reply:
Ok fixed. Thanks!
I have gotten mixed messages about that namespace prefix :)